notebooks/en/Prompt_Tuning_PEFT.ipynb

```diff
@@ -0,0 +1,7324 @@
+{
```

Comment:
<p>Instead of this, you can do the below which would avoid loading a second copy of the base model in memory.</p><pre class="ql-syntax" spellcheck="false">loaded_model_prompt.load_adapter(output_directory_sentences, adapter_name="quotes")
loaded_model_prompt.set_adapter("quotes")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Amazing! Thaks I changed the code! </p>